src/libraries/System.Text.Json/src/System/Text/Json/Serialization/Metadata/ReflectionJsonTypeInfoOfT.cs

```diff
@@ -10,88 +10,87 @@
 namespace System.Text.Json.Serialization.Metadata
```

Comment:
Note that I didn't rename either `ReflectionJsonTypeInfoOfT.cs` nor `SourceGenJsonTypeInfoOfT.cs` for fear of destroying the diffs. I'll follow up with a commit that renames these appropriately.